packages/rich-text/src/plugins/Quote/toggleQuote.ts

```diff
@@ -4,26 +4,35 @@ import isHotkey from 'is-hotkey';
 import { Transforms, Element, Editor } from 'slate';
 
 import { isBlockSelected } from '../../helpers/editor';
+import { CustomElement } from '../../types';
 
 export function toggleQuote(editor: PlateEditor): void {
   if (!editor.selection) return;
 
   const isActive = isBlockSelected(editor, BLOCKS.QUOTE);
 
   Editor.withoutNormalizing(editor, () => {
+    if (!editor.selection) return;
+
     Transforms.unwrapNodes(editor, {
-      match: (node) => Element.isElement(node) && node.type === BLOCKS.QUOTE,
+      match: (node) => Element.isElement(node) && (node as CustomElement).type === BLOCKS.QUOTE,
       split: true,
     });
 
+    const { anchor, focus } = editor.selection;
+    const isTripleSelection =
+      anchor.path[0] !== focus.path[0] && anchor.offset === 0 && focus.offset === 0;
```

Comment:
How sure are we of this check? is it the recommended way to check for a triple select in slate?

Reply:
For example, when there's a triple selection, `focus.path[0]` will always `anchor.path[0] + 1`, so it always get the next path.

Valid case:
```js
{
  anchor: { path: [1, 0], offset: 0 },
  focus: { path: [2, 0], offset: 0 } 
}
```

So we can see that `.focus` selects the next element and the `offset` is set to `0` because the selection starts at the beginning of both elements. That's why I'm not using `.focus` here [`f17c421` (#1021)](https://github.com/contentful/field-editors/pull/1021/commits/f17c4213dffcd546138ce09ec3599cbd80e2e143#diff-4b9fbdb58a22376a73c74bceb917c863f1c56d9e02c096332f77a63cbae48358R75)